pennylane_sf/simulator.py

```diff
@@ -182,3 +184,46 @@ def observables(self):
             set[str]: the set of PennyLane observable names the device supports
         """
         return set(self._observable_map.keys())
+
+    def probability(self, wires=None):
+        """Return the (marginal) probability of each computational basis
+        state from the last run of the device.
+
+        Args:
+            wires (Sequence[int]): Sequence of wires to return
+                marginal probabilities for. Wires not provided
+                are traced out of the system.
+
+        Returns:
+            OrderedDict[tuple, float]: Dictionary mapping a tuple representing the state
+            to the resulting probability. The dictionary should be sorted such that the
+            state tuples are in lexicographical order.
+        """
+        N = len(wires)
+        cutoff = getattr(self, "cutoff", 10)
```

Comment:
Do we need to add the `cutoff` attribute to `StrawberryFieldsGaussian` with a default?

Reply:
Yep we do!

Reply:
Just haven't had a chance yet haha